uvloop/loop.pyx

```diff
@@ -216,7 +215,7 @@ cdef class Loop:
         self._servers = set()
 
     cdef inline _is_main_thread(self):
-        return MAIN_THREAD_ID == PyThread_get_thread_ident()
+        return threading_main_thread().ident == PyThread_get_thread_ident()
```

Comment:
@fantix this is a pretty performance-critical function. I don't like calling `threading_main_thread()` here. We should instead cache it on the first call.

Reply:
I think so too.

Even though the code change makes sense, I'm still trying to reproduce the original issue with `multiprocessing`. It just seems it's a very specific mixing usage of uvloop and `multiprocessing` - I'll add a test if reproduced.